### PR TITLE
Document system welcome messages for CLAN servers

### PR DESCRIPTION
**Pull request by owocado**

### CODE REVIEW

File: pages/resources/message.mdx [hunk 1]
```diff
@@ -139,6 +143,19 @@ The type of rendered message is determined via converting the message's `timesta
 | 11    | "Good to see you, {author}."                    |
 | 12    | "Yay you made it, {author}!"                    |
```
[the remark this answers is not in the document]
Comment (owocado) [reply pages/resources/message.mdx:145]: what should the new heading be called

[the remark this answers is not in the document]
Comment (owocado) [reply pages/resources/message.mdx:145]: if we add new heading, this section in https://docs.discord.sex/resources/message#message-type will also require new jump link for clarification
image

File: pages/resources/message.mdx [hunk 2]
```diff
@@ -123,6 +123,10 @@ Type `19` and `20` are only available in API v8 and above. In v7 and below, they
 
 The type of rendered message is determined via converting the message's `timestamp` to a unix timestamp with millisecond precision, modulo 13.
 
+However, if the guild has `CLAN` feature, the same is determined via dividing the message's `timestamp` with millisecond precision by modulo 10.
+
+See the 2nd table below for system welcome messages that shows up in CLAN servers.
```
Comment (dolfies) [inline pages/resources/message.mdx:128]:
```suggestion
However, if the guild has the [`CLAN` feature](/resources/guild#guild-features), the same is determined via dividing the message's `timestamp` with millisecond precision by modulo 10.

See the 2nd table below for the system welcome messages specific to clans.
```

File: pages/resources/message.mdx [hunk 3]
```diff
@@ -123,6 +123,10 @@ Type `19` and `20` are only available in API v8 and above. In v7 and below, they
 
 The type of rendered message is determined via converting the message's `timestamp` to a unix timestamp with millisecond precision, modulo 13.
```
Comment (dolfies) [inline pages/resources/message.mdx:124]:
```suggestion
 Usually, the type of rendered message is determined via converting the message's `timestamp` to a unix timestamp with millisecond precision, modulo 13.
```

**Review by dolfies — CHANGES_REQUESTED** [review]

File: pages/resources/message.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (dolfies) [reply pages/resources/message.mdx:145]: Yup, moving the new clan welcome message description in between the tables would be perfect

[the remark this answers is not in the document]
Comment (owocado) [reply pages/resources/message.mdx:141]: ok done, how does it look now :3

**Review by dolfies — APPROVED** [review]

**Comment by dolfies** [comment]
ty!

**Pull request merged**